.gitignore

```diff
@@ -152,3 +152,4 @@ workers/spdx_worker/dosocs2.conf
 workers/spdx_worker/augur-spdx/
 workers/spdx_worker/ex-raw.txt
 workers/spdx_worker/ex.json
+backup_test
```

Comment:
Was this intentional? I'm not familiar with the SPDX worker, but I'm also not sure how this fits into the rest of the PR.

Reply:
It's by mistake.